code/datums/elements/waddling.dm

```diff
@@ -16,9 +16,9 @@
 /datum/element/waddling/proc/LivingWaddle(mob/living/target)
 	if(target.incapacitated() || !(target.mobility_flags & MOBILITY_STAND))
 		return
-	Waddle()
+	Waddle(target)
 
 /datum/element/waddling/proc/Waddle(atom/movable/target)
 	animate(target, pixel_z = 4, time = 0)
-	animate(target, pixel_z = 0, transform = turn(matrix(), pick(-12, 0, 12)), time=2)
-	animate(target, pixel_z = 0, transform = matrix(), time = 0)
+	animate(pixel_z = 0, transform = turn(matrix(), pick(-12, 0, 12)), time=2)
```

Comment:
isn't this going to animate the datum?

Reply:
Yes, you are supposed to leave the object out for following animation steps. Its a bit weird, but its in the ref. Stating target for all three broke the animation.

Reply:
I mistook it and thought empty first arg meant src by default but nope it's just weird